experiment/procedure.md

```diff
@@ -1 +1,13 @@
-### Procedure
\ No newline at end of file
+ 
+**Procedure**
+
+1. Dimensions of test piece is measured at 3 different places along its height/length to determine the average c/s area.
+2. Ends of the specimen should be plane. For that the ends are tested on a bearing plate.
+3. The specimen is placed centrally between the two compression plates, such that the center of moving head is vertically above the center of specimen.
+4. Load is applied on the specimen by moving the movable head.
+5. The load and the corresponding contraction are measured at different intervals. The load interval may be as 500kg.
+6. Load is applied until the specimen fails.
+7. The values of stress and strain are calculated at several intervals and are plotted on a graph.
+8. The resultant graph is the Stress Strain curve which is the property of a material and is independent of the dimensions of specimen used. 
```

Comment:
Please use a hyphen - between Stress and Strain to make it look like the graph is plotted between these 2 parameters.